test/testutils/proto_compare.go

```diff
@@ -20,5 +20,5 @@ func ExpectEqualProtoMessages(a, b proto.Message) {
 	// One shortcoming is that you only get +/- 5 chars of context
 	// per: https://github.com/onsi/gomega/blob/master/format/format.go#L146
 	// TODO(mitchdraft) gomega pr to modify charactersAroundMismatchToInclude (if not merged will make a util)
-	Expect(a.String()).To(Equal(b.String()))
+	ExpectWithOffset(1, a.String()).To(Equal(b.String()))
```

Comment:
You don't need this since we have `PrintTrimmedStackTrace` in the fail handers (see the suite file)

~Also, this util was moved to go-utils, could you delete it here and use_~
nvm - has not merged yet https://github.com/solo-io/go-utils/pull/149
anyway, no need to update it here

Reply:
Nice feature! I did not notice it, will switch back